### PR TITLE
Limit selection of translations (#718)

### LINKED ISSUE #718: Limit selection of translations: User should not be able select more than xx translations
Maybe max 10 translations.

### PR DESCRIPTION
**Pull request by azeezolaniran2016**
# Title:
Fixes #718

## Description:
Limits the number of translations selectable to a maximum of 10.

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32813

File: src/components/ContentDropdown/index.js [hunk 1]
```diff
@@ -10,45 +10,52 @@ import Icon from 'quran-components/lib/Icon';
...
-    return 0;
-  };
+  return 0;
+};
 
+const TRANSLATIONS_SELECTION_LIMIT = 10;
 
 class ContentDropdown extends Component {
+  state = {
+    selectedTranslationsCount: this.props.translations.length
```
Comment (azeezolaniran2016) [inline src/components/ContentDropdown/index.js]: Thanks @mmahalwy. I've updated it.

**Comment by mmahalwy** [comment]
thanks @azeezolaniran2016 !

**Comment by mmahalwy** [comment]
rebuild

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32822

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32826

**Pull request merged**